runfolder_tests/unit/runfolder_tests.py

```diff
@@ -110,6 +110,24 @@ def test_get_library_tube_barcode_found(self):
         # Test
         self.assertEqual(runfolder_svc.get_library_tube_barcode('/path/to/runfolder/', runparameters_dict), 'NV0012345-LIB')
 
+    def test_get_novaseqxplus_library_tube_barcode_found(self):
+        # Setup
+        configuration_svc = dict()
+        configuration_svc["monitored_directories"] = ["/data/testarteria1/runfolders"]
+        runfolder_svc = RunfolderService(configuration_svc, logger)
+        runparameters_dict = {
+                                "RunParameters": {
+                                    "ConsumableInfo": {
+                                        "ConsumableInfo": [
+                                            {"Type": "SampleTube", "SerialNumber": "LC1037822-LC1"}
```

Comment:
Not really necessary, but to make sure that the function takes the right ConsumableInfo and not just the first, I would add another Consumable Info at the start of the list. Something like: 
```
[
    {"Type": "Other", "SerialNumber": "INCORRECT123"},
    {"Type": "SampleTube", "SerialNumber": "LC1037822-LC1"},
]

```

Reply:
Yes, sure, I agree with that. 

I have now made this change.